src/python/pants/engine/legacy/graph.py

```diff
@@ -82,7 +77,8 @@ class _DestWrapper:
 class LegacyBuildGraph(BuildGraph):
   """A directed acyclic graph of Targets and dependencies. Not necessarily connected.
 
-  This implementation is backed by a Scheduler that is able to resolve TransitiveHydratedTargets.
+  This implementation is backed by a Scheduler that is able to resolve
+  LegacyTransitiveHydratedTargets.
```

Comment:
Could you expand the description of #4535 to account for this new layer of indirection?